src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -109,12 +109,16 @@
                 }
             </div>
         </div>
-        <div class="float-right">
-            <form asp-resource="@Model.Review" class="form-inline float-left ml-2" asp-page-handler="ToggleApproval" method="post" asp-requirement="@ApproverRequirement.Instance">
-                <input type="hidden" name="revisionId" value="@Model.Revision.RevisionId" />
-                <input type="submit" class="btn btn-sm btn-secondary" value="@(Model.Revision.Approvers.Contains(User.GetGitHubLogin()) ? "Revert Approval" : "Approve")" />
-            </form>
-        </div>
+        @if (Model.DiffRevision == null || Model.DiffRevision.Approvers.Count > 0)

```

Comment:
I would disable the button and add a tooltip instead. It would be also nice to show in the dropdown of diff revisions which ones are approved (maybe have them be green?)

Reply:
I have made these change. Below is screenshot of revision dropdown with approved revisions in green color.


![image](https://user-images.githubusercontent.com/55455725/123031535-69f16800-d3b2-11eb-95f8-1733803ad7f1.png)